xbmc/guilib/GUIIncludes.cpp

```diff
@@ -295,6 +300,127 @@ void CGUIIncludes::ResolveIncludesForNode(TiXmlElement *node, std::map<INFO::Inf
     node->FirstChild()->SetValue(ResolveConstant(node->FirstChild()->ValueStr()));
 }
 
+map<string, string> CGUIIncludes::GetParameters(const TiXmlElement *includeCall, const TiXmlElement *includeDef) const
+{
+  static const char paramNamespacePrefix[] = "p:";
+  map<string, string> params;
+  const TiXmlAttribute *attribute;
+  // add actual parameters from include call tag (these override defaults)
+  // <include file="MyControls.xml" p:posx="225" p:posy="150">MyControl1</include>
+  if (includeCall)
+  {
+    attribute = includeCall->FirstAttribute();
+    while (attribute)
+    {
+      if (StringUtils::StartsWith(attribute->Name(), paramNamespacePrefix))
```

Comment:
You need to make sure that the return value of `attribute->Name()` is not NULL because `StringUtils::StartsWith()` doesn't do that for you and simply tries to dereference it.